Assets/MixedRealityToolkit.Tests/EditModeTests/InputSystem/TestFixture_03_InputSystemTests.cs

```diff
@@ -78,7 +78,7 @@ public void Test04_TestMixedRealityInputSystemExists()
         [TearDown]
         public void CleanupMixedRealityToolkitTests()
         {
-            TestUtilities.CleanupScene();
+            TestUtilities.CreateScenes();
```

Comment:
Do you also need to tear down the scenes in the playmode tests? Interesting that there are only two locations where we create / tear down scenes in edito mode tests.

Reply:
Methods marked with [TearDown] get called after each test is run so it's destroying scene assets every time.